backend/apps/packages/services.py

```
@@ -192,6 +193,16 @@ def sync_package_snapshots(package: Package, resident: Membership) -> None:
     package.building_snapshot = bedroom.edificio if bedroom else ""
 
 
+def _validate_and_save_package(package: Package) -> Package:
+    try:
+        package.full_clean()
+    except DjangoValidationError as exc:
+        raise ValidationError(exc.message_dict or exc.messages)
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)